aim2dat/strct/ext_manipulation/add_structure.py

```diff
@@ -284,7 +280,12 @@ def add_structure_coord(
 def _check_guest_structure(guest_strct: Union[Structure, str]) -> dict:
     if isinstance(guest_strct, Structure):
         label = "" if guest_strct.label is None else guest_strct.label
-        return {"elements": guest_strct.elements, "positions": guest_strct.positions}, label
+        return {
+            "elements": guest_strct.elements,
+            "positions": guest_strct.positions,
+            "kinds": [None] * len(guest_strct) if guest_strct.kinds is None else guest_strct.kinds,
```

Comment:
As a general comment, have we already considered to return `[None] * len(elements)` in case `kinds` is not set in a `Structure` object? We always have these if-else statements and I was wondering if there is a reason against this kind of default value for `kinds`? It is also handled in a similar way in `ASE`: https://wiki.fysik.dtu.dk/ase/_modules/ase/atoms.html#Atoms.get_tags. Not to say that we have to do it like that, but just that this might be something to consider

For sure, this would require a revision of all the structural methods and doesn't have to be done right now, I just wanted to put this topic on the table.

Reply:
This is a good point. I opened an issue (#66) and will address this still before the next release.

Reply:
Great!